tools/etc/frr/support_bundle_commands.conf

```diff
@@ -153,3 +153,41 @@ show ip pim state
 show ip pim statistics
 show ip pim rpf
 CMD_LIST_END
+
+# OSPFv3 Support Bundle Command List
+PROC_NAME:ospf6
+CMD_LIST_START
+show ipv6 ospf6
+do show ipv6 ospf6 border-routers
+do show ipv6 ospf6 border-routers detail
+show ipv6 ospf6 database
+show ipv6 ospf6 database detail
+show ipv6 ospf6 database dump
+show ipv6 ospf6 database internal
+show ipv6 ospf6 database router detail
+show ipv6 ospf6 database network detail
+show ipv6 ospf6 database inter-prefix detail
+show ipv6 ospf6 database inter-router detail
+show ipv6 ospf6 database intra-prefix detail
+show ipv6 ospf6 database link detail
+show ipv6 ospf6 database as-external detail
+show ipv6 ospf6 database self-originate detail
+show ipv6 ospf6 database type-7 detail
+show ipv6 ospf6 interface
+show ipv6 ospf6 interface prefix
+show ipv6 ospf6 interface traffic
+show ipv6 ospf6 linkstate detail
+show ipv6 ospf6 neighbor
+show ipv6 ospf6 neighbor drchoice
+show ipv6 ospf6 neighbor detail
```

Comment:
we need both neighbor and neighbor detail?

Reply:
I have kept both considering that it is easier to first look into the neighbor output and then detail if required.

Reply:
Having just used this output (and the normal + detail in this way), this is at least compelling enough of a reason to not block the PR since we have quite a few OSPF failures recently in CI.

Reply:
I have another PR open which I will adjust to include a couple missing command now that this is merged and if @donaldsharp you feel strongly about not having the brief and detail commands I can remove them there.

Reply:
@donaldsharp please see #9341 if you would like to request their removal